### ISSUE TITLE
API Version 3

### ISSUE THREAD

**Issue by fliese** [robotmedia/AndroidBillingLibrary#113]
Hey, 
will you integrate Android Billing API Version 3 to the project?

**Comment by lakeba** [comment]
Interesting question.....

**Comment by hpique** [comment]
Yes. I expect to do it sometime in February. Feel free to start without me. ;)

**Comment by foodtruckfiesta** [comment]
Just fyi, 3 days ago, Google announced V3 support for subscriptions.  http://developer.android.com/google/play/billing/billing_subscriptions.html

**Comment by jberkel** [comment]
FYI, i cleaned up, tested and packaged Google's IAB v3 sample code as an apklib.
https://github.com/jberkel/pay-me

contributions welcome

**Comment by serso** [comment]
Guys, there is already a library which works with Billing Version 3 and supports fallback for users who can't use it (but who might have purchases in Billing Version 2). And the library is called Checkout: https://github.com/serso/android-checkout. Please, read the README to know how to turn on the fallback functionality.
